### ISSUE TITLE
Button delete doesn't work in Mobile interface

### ISSUE THREAD

**Issue by anton01** [btactic/zimbra-catala#5]
Hello in mobile interfice it's not possible delete mails.

Server Zimbra 8.6 patch 6.

To reproduce this issue you should:
· 1st Log in mobile webmail.
· 2nd select mail.
· 3rd Push button ·Accions· -> select "Eliminar".

After this the string button change from "Accions" to "Eliminar", the mail has not deleted.

**Comment by anton01** [comment]
Hello.
I want correct the issue.
Real problem it's that action button doesn't work  when you are logged in webmail from smartphone.

**Comment by anton01** [comment]
Hello.

I did a debug of web, I found next error clicking over "Eliminar" button.
`An error occured in: https://172.26.0.80/m/zmain
{
    msg: Uncaught ReferenceError: submitForm is not defined,
    line: 712
}
zmain:712 Uncaught ReferenceError: submitForm is not definedonchange @ zmain:712
`

The line 712 of file Zmain have next  code:
`<select class="zo_select_button" name="anAction" onchange="return submitForm(document.getElementById('zForm'),null,this.value);">`

I've checked all catalan lenguage file, but apparently doesn't missing any item.

Any idea
